sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/TemplateSpecVersionsOperations.cs

```diff
@@ -253,11 +253,11 @@ public async Task<AzureOperationResponse<TemplateSpecVersion>> CreateOrUpdateWit
             string _responseContent = null;
             if ((int)_statusCode != 200 && (int)_statusCode != 201)
             {
-                var ex = new DefaultErrorResponseException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
+                var ex = new TemplateSpecsErrorException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
```

Comment:
Wouldn't this be a breaking change if any customer is catching DefaultErrorResponseException?

Reply:
Echoing @stuartko: The Template Specs service is currently only an opt-in service for a select few private preview customers. We don’t anticipate that any customers will be using the SDK prior to us hitting Public Preview later this month, so there shouldn’t be any issue with the exception rename adversely impacting anyone.